src/services/validators/validate-cron.js

```diff
@@ -28,7 +28,7 @@ const isValidNumberRange = (range, x, y) => {
     return (
         isValidNumber(boundaries[0], x, y) &&
         isValidNumber(boundaries[1], x, y) &&
-        boundaries[0] <= boundaries[1]
+        Number(boundaries[0]) <= Number(boundaries[1])
```

Comment:
since these have already been evaluated as valid numbers, we should be able to use `Number()` here